services/ml/execution_impl_android.cc

```diff
@@ -41,38 +42,42 @@ ExecutionImplAndroid::ExecutionImplAndroid(
         offset, length, memory_->MapAtOffset(length, offset)));
     total_length += length;
   }
+}
 
-  int32_t result = ANeuralNetworksExecution_create(compilation->nn_compilation_,
-                                                   &nn_execution_);
+void ExecutionImplAndroid::PrepareForCompute(
+    ANeuralNetworksExecution*& execution) {
```

Comment:
What does this ANeuralNetworksExecution`*&`  mean?

Reply:
> What does this ANeuralNetworksExecution`*&` mean?

Here I wanted to use a pointer type quotation for  the new added function. Now I have deleted the new function added. I have updated these codes following your suggestions, please help to review,thanks! @fujunwei